tests/test_pos_mev_enabled.py

```diff
@@ -155,14 +158,70 @@ def test_validator_exited(tilt_up):
     with open(f"{os.path.dirname(__file__)}/deposit_data.json") as f:
         deposit_data = json.load(f)
 
-    # Send exits
-    for i, private_key in enumerate(deposit_data["private_keys"]):
+    # Offline preparation is necessary, since
+    # ethdo does not support minimal beacon spec, making it unable
+    # to read necessary beacon data due to ssz encoding mismatches
+    beacon_genesis_data = requests.get(f"{base_url}/eth/v1/beacon/genesis").json()
+    beacon_fork_data = requests.get(f"{base_url}/eth/v1/beacon/states/head/fork").json()
+    beacon_spec = requests.get(f"{base_url}/eth/v1/config/spec").json()
+
+    # Fork version: honor EIP-7044
+    current_fork_version = beacon_fork_data["data"]["current_version"]
+    if current_fork_version >= beacon_spec["data"]["DENEB_FORK_VERSION"]:
```

Comment:
What is the if for here? Shouldn’t domain of exit always be capella?

Reply:
It's possible to configure `eth-possim` to start from other fork, then this logic will be correct

Reply:
why would one do that?

Reply:
Test other forks apart from Deneb